README.md

```diff
@@ -1,5 +1,7 @@
+[![Build Status](https://travis-ci.org/andela/ah-code-blooded.svg?branch=develop)](https://travis-ci.org/andela/ah-code-blooded)
```

Comment:
From what branch did you check out from? This also doesn't seem to be related to this task.

Reply:
Given that the branch with the PR was not from develop, I checked out develop and created another branch ft-signup-validators-160577607 to house the code for the PT story. Therefore, this branch should be discarded because it does not use develop as the base. I created a local branch and later created an upstream for it. I have created a new branch that appropriately uses develop as the base. Thanks